src/components/frame/FrameConnector.tsx

```diff
@@ -105,14 +105,23 @@ export const FrameConnector: FunctionComponent<FrameConnectorProps> = ({
     }
   }, [connected, toFrame, onConnected, onConnectedCalled]);
   return (
-    <iframe
-      title="Decentralised Rendered Certificate"
-      id="iframe"
-      ref={iframe}
-      src={source}
-      style={style}
-      className={className}
-      sandbox={sandbox}
-    />
+    <>
+      {timeout ? (
```

Comment:
I'm fine with this but I think the idea is also to send a message to the verifier, so that it is notified that something wrong happened and can decide to react accordingly (either by showing a custom error or leaving the frame as is